pages/categories/operational.md

```diff
@@ -41,10 +41,63 @@ As the organisation does not own or control the IT assets delivering these servi
+
+{% include linkedHeading.html heading="Environmental impact of Cloud Services" level=4 %}
+
+The environmental impact and carbon emissions associated with cloud computing are becoming increasingly important considerations. The energy consumption and carbon footprint of cloud data centres have grown substantially as the adoption of these services has accelerated. This hardware has its own Upstream and Operational emissions, which are attributable to clients under Scope 3 of the GHG protocol. Despite improvements in energy efficiency, the sheer scale of cloud infrastructure means its collective emissions are significant.
+
+At the same time, cloud computing presents some opportunities for improved sustainability compared to traditional on-premises data centres. Cloud providers can leverage economies of scale to invest in more energy-efficient hardware, cooling systems, and renewable energy sources. The flexibility of the cloud also allows organisations to dynamically provision resources and optimise their usage, potentially reducing idle capacity and associated emissions.
+
+Focussing specifically on energy efficiency, Cloud providers often use custom hardware with a lower power demand. Generally speaking, it is in their interests to use the most energy efficient hardware possible to reduce their own operating costs. This also allows users to switch to lower powered hardware without the initial outlay for new hardware. Improved energy efficiency also extends to the typical [Power Usage Effectiveness (PUE)](/glossary#power-usage-effectiveness-pue) of a cloud based data centre, which can be lower than typical on-premise data centres.
+
+| Provider                      | PUE        |
+| ----------------------------- | ---------- |
+| Alibaba Cloud                 | 1.3        |
+| AWS                           | 1.135      |
+| Azure                         | 1.185      |
+| GCP                           | 1.1        |
+| Oracle Cloud                  | 1.15       |
+| Worldwide Data centre Average | 1.58 - 1.8 |
+
+Sources: [^ccf-pue] [^oracle-pue]
+
+[^ccf-pue]: [Cloud Carbon Footprint Methodology](https://www.cloudcarbonfootprint.org/docs/methodology#appendix-i-energy-coefficients)
+[^oracle-pue]: [Oracle Climate Action](https://www.oracle.com/uk/government/climate-action/#rc30p2)
+
+These figures should be considered carefully when sourced from the Cloud Providers themselves, as it may apply to specific data centres only rather than being an average. In future it is hoped that these figures can be further reduced by recapturing the waste heat from data centres, with [Deep Green](https://deepgreen.energy/) being an example of a smaller Cloud provider whose micro data centres extract heat to provide hot water for a range of purposes.
```

Comment:
> it may apply to specific data centres only rather than being an average

For my understanding, does this mean that the figures supplied by Cloud providers for their own facilities might only apply to specific Cloud data centres?

Reply:
I think Cloud providers are usually clear about what the figures are but they may not provide an average. Oracle for example I couldn't find an average figure, so the source for theirs is:
>  Oracle data centres worldwide have also achieved power usage effectiveness (PUE) **_as low as 1.15_**